mysql-test/suite/versioning/t/trx_id.test

```diff
@@ -411,3 +411,54 @@ DROP TABLE tts;
 DROP TABLE ttx;
 DROP FUNCTION fts;
 DROP FUNCTION ftx;
+
+--echo #
+--echo # MDEV-16330 Allow instant change of WITH SYSTEM VERSIONING column attribute
+--echo #
+
+SET @@SYSTEM_VERSIONING_ALTER_HISTORY=KEEP;
+CREATE TABLE t (
+  a INT,
+  b INT,
+  row_start BIGINT UNSIGNED AS ROW START INVISIBLE,
+  row_end BIGINT UNSIGNED AS ROW END INVISIBLE,
+  PERIOD FOR SYSTEM_TIME(row_start, row_end)
+) WITH SYSTEM VERSIONING ENGINE=INNODB;
+
+INSERT INTO t VALUES (1,1);
+
+--echo without table rebuild
+ALTER TABLE t
+  CHANGE a a INT WITHOUT SYSTEM VERSIONING;
```

Comment:
Add `--enable_info` so that if the operation is executed using `ALGORITM=INPLACE` (as we expect), then 0 rows affected will be reported, instead of the actual number of rows that `ALGORITHM=COPY` would report.